### PR TITLE
[INFRA] Simulate automate tag and files update of bpmn-visualization-R

### PR DESCRIPTION
**Pull request by csouchet**
- Add the information of https://github.com/process-analytics/bpmn-visualization-R to update in a workflow to test
- Rename `release.yml` in `release-bpmn_visualization.yml`
- Add new workflow for the relesae of the R package: `release-R.yml`
  - with a step to unprotect the branch
  - with a step to get the current version
  - with a step to generate new version with https://github.com/marketplace/actions/generate-semantic-version
  - with a step to update README.md with the release version
  - with a step to update DESCRIPTION with the release version
  - with a step to commit
  - with a step to tag
  - with a step to push the tag
  - with a step to update DESCRIPTION with the development version
  - with a step to protect the branch

covers https://github.com/process-analytics/bpmn-visualization-R/issues/47

### CODE REVIEW

File: .github/workflows/release-R.yml [hunk 1]
```diff
@@ -21,10 +21,6 @@ jobs:
           enforce_admins: false
           branch: master
 
-      - name: Setup node
-        uses: actions/setup-node@v3
-        with:
-          node-version: '16'
```
Comment (tbouffard) [inline .github/workflows/release-R.yml:27]: must be kept as we also running `npm version`

File: README.md [hunk 2]
```diff
@@ -38,3 +38,13 @@ The event is received by the repository which triggers workflows (simulate what
 ## Extra
 
 - [Dependabot configuration](.github/dependabot.yml): as configured in repository using dependabot to update dependencies
+
+
+## Test to update the readme of https://github.com/process-analytics/bpmn-visualization-R
```
Comment (tbouffard) [inline README.md:43]: I suggest we found a better wording that explain that the following paragraph is use to mimic the R package README. 
It was not obvious to me at a first glance that the content was added for this purpose

File: .github/workflows/release-R.yml [hunk 3]
```diff
@@ -1,4 +1,4 @@
-name: Release
```
[the remark this answers is not in the document]
Comment (csouchet) [reply .github/workflows/release-R.yml]: I don't know what it happened, but I created a copy for the R release. I don't know why the file for bpmn-visu disappeared.

File: .github/workflows/release-R.yml (hunk 1 above)
Comment (csouchet) [reply .github/workflows/release-R.yml]: No need of npm or node for R project.

File: README.md [hunk 4]
```diff
@@ -38,3 +44,15 @@ The event is received by the repository which triggers workflows (simulate what
...
```
Comment (tbouffard) [inline README.md:49]:
```suggestion
## Test to update the README of https://github.com/process-analytics/bpmn-visualization-R
```

File: README.md [hunk 5]
```diff
@@ -38,3 +44,15 @@ The event is received by the repository which triggers workflows (simulate what
...
+
+The following simulates the content of the `README.md` file of [bpmn-visualization-R](https://github.com/process-analytics/bpmn-visualization-R) repository.
```
Comment (tbouffard) [inline README.md:51]:
```suggestion
The following simulates the content of the `README.md` file of [bpmn-visualization-R](https://github.com/process-analytics/bpmn-visualization-R) repository. This content is updated automatically during the release process.
```

File: .github/workflows/release-R.yml [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+
+      - name: Update DESCRIPTION with the release version
+        run: |-
+          sed -i -E 's/Version: "(.*)"/Version: "${{ steps.release_version.outputs.version }}"/g' DESCRIPTION
+
+      - name: Commit
+        run: |
+          git add README.md
+          git add DESCRIPTION
+          git commit -m "[RELEASE] Prepare version for release"
```
Comment (tbouffard) [inline .github/workflows/release-R.yml:65]: ⚠️ commit message to check while testing

File: .github/workflows/release-R.yml (hunk 3 above)
[the remark this answers is not in the document]
Comment (tbouffard) [reply .github/workflows/release-R.yml]: fixed

**Review by tbouffard — APPROVED** [review]
LGTM, let's test!

**Pull request merged**